src/FMBot.Bot/Builders/FriendBuilders.cs

```diff
@@ -388,4 +389,56 @@ public async Task<ResponseModel> RemoveFriendsAsync(ContextModel context, string
 
         return response;
     }
+
+    public async Task<ResponseModel> FriendedAsync(ContextModel context)
+    {
+        var response = new ResponseModel
+        {
+            ResponseType = ResponseType.Embed,
+        };
+
+        if (context.DiscordGuild != null)
+        {
+            response.Embed.WithDescription("This command is only supported in DM");
+            response.CommandResponse = CommandResponse.Error; // unsure if some other type should be used
```

Comment:
If you want you can make a new `CommandResponse` type just for this. There is already a similar one for when commands aren't supported in DM.

Reply:
i've assumed this is as simple as adding the type and that there's no surrounding things to consider.